packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.spec.tsx

```diff
@@ -107,6 +111,7 @@ describe('<SampleTypePropertiesPanel/>', () => {
                     description: descVal,
                 }),
                 domainKindName: 'SampleType',
+                domainDesign: DomainDesign.create({ allowTimepointProperties: false }),
```

Comment:
could you use the `sampleTypeModel` const you created above here? something like `domainDesign: sampleTypeModel.domainDesign`?

Reply:
Good point, done!